xgboost/gbt.py

```diff
@@ -65,6 +65,12 @@ def convert_xgb_predictions(y_pred, objective):
                     choices=('reg:squarederror', 'binary:logistic',
                              'multi:softmax', 'multi:softprob'),
                     help='Control a balance of positive and negative weights')
+parser.add_argument('--count-dmatrix', default=False, action='store_true',
+                    help='Count DMatrix creation in time measurements')
+parser.add_argument('--single-precision-histogram', default=False, action='store_true',
+                    help='Build histograms instead of double precision')
+parser.add_argument('--enable-experimental-json-serialization', default=True,
```

Comment:
`default=False` is better if this feature affects perf.

Reply:
It's true by default in XGBoost, so, as we discussed, I decided to leave it as is